financials/spending_log.csv

```diff
@@ -20,3 +20,6 @@ spend_id,date,spend_description,decision_log_id,currency,amount,exchange_rate,US
 19,20200107,yeastplume Q1 2020 funding,49,BTC,4.5,7768.23,34957.04,bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx,3MWCNS6JUDZkpAnBiB8XfyaJQaBuQn19yH,5beb648dd9829b19afc755c5b920cd331b4b1c3c2b1b43382bb53faefdd63e8b,full payment
 20,20200115,lehnberg Q1 2020 funding,52,BTC,1.10362282,8807.01,9719.62,39hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4,1DUeRhn3CuhFX9ygNsEmRVLNmsh2aiju1q,385f1236b7a56a054dd60a393528af733efd714939aa6cdabcf73adf9a969a6f,full payment
 21,20200228,jaspervdm Mar-May 2020 funding,57,BTC,3.80615000,8672.46,33008.68,bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx,39FQvK7jySjDEJyU6xWYgnC4qiJyTzbpsk,9910a337cba3079bd6eca40db593798fcde35cb87b8a3a3abd45ece3d920d43b,full payment
+22,20200402,lehnberg Q2 2020 funding,59,BTC,1.50,6793.62,10190.43,bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx,1N6S5xxMLhV1QTNZX8iakfPxyun3zJWQcY,4bf828130eb9da400e09ff304a14d8385c338141d28bf2732738f3d8e75aa2ae,full payment
+23,20200409,yeastplume Q2 2020 funding,58,BTC,4.37,7302.09,31910.14,bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx,37NWmCVUEmBWwx1VFb1Q2boYmtza6kRCYw,8d9c9dfd0edb6175ccea1c0f7b52d194e702daeb9b340defb13c5a7faff64eba,full payment
+24,20200409,antiochp Q2 2020 funding,61,BTC,4.51,7302.09,32932.43,bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx,bc1q6q08k8n0k6cgl6asq8gekpxntxuu5h3p2stlr3,7a695b7a61d7df006b20bdcb7e29e9f79fca991d4e41a045418328acbe2da5f3,full payment

```

Comment:
Are you sure about the exchange rates here?

Reply:
Yes. They are the closing rates from https://coinmarketcap.com/currencies/bitcoin/historical-data/

Related: https://forum.grin.mw/t/correcting-my-funding-discrepancies/7242/

Reply:
Also note that this is USD rates, whilst my request is in GBP and Antioch's and Yeast's is in EUR.

Reply:
Ah sorry mixed up USD/EUR...